pylib_3.9.7/ruse/bio/bio_data_table_helper.py

```diff
@@ -71,6 +71,8 @@ def genbank_base64_str_to_sequence(data: str, row_index: int) -> SeqRecord:
             genbank_str = value.decode(charset)
             with StringIO(genbank_str) as fh:
                 record = SeqIO.read(fh, 'gb')
+            if record.id == '':

```

Comment:
Assuming record.is is a string, shouldn't this be

```python
if not record.id:
```

Reply:
Nice catch!  I never think about the "falseness" of the empty string.  👍